### ISSUE TITLE
[Question] No __version__ attribute in apex package

### ISSUE THREAD

**Issue by vfdev-5** [NVIDIA/apex#337]
Hi, I was just wondering why there is no `__version__` in apex package to check the version (even 0.1).
Thanks

**Comment by ptrblck** [comment]
Hi @vfdev-5!
I think we should definitely add versioning to make debugging easier and to track new features in `apex`.
Thanks for the pointer! :)

**Comment by h-vetinari** [comment]
Yes, please tag a release, and even better: put the package on PyPi. :)
